client/src/templates/Introduction/intro.css

```diff
@@ -426,8 +427,10 @@ button.map-title {
   padding: 25px;
 }
 
+/* this margin is used to center the element. 
+ToDo: find out why, and remove the need for it */
 .block-grid .map-title > svg {
-  margin: 10px;
+  margin-bottom: 4px;
```

Comment:
Not sure about this. We can do margin: 4px if you wanna save space.

Reply:
ya this one is weird, I am using it to center the icon with the title in `/learn/{courseSlug}`, the test below happened in `/learn/2022/responsive-web-design/`


with `margin-bottom: 4px`
![image](https://user-images.githubusercontent.com/88248797/201135063-c3d5050e-3cc2-42f2-bee7-7f249fabd666.png)


Here how it looks with `margin: 4px`

![image](https://user-images.githubusercontent.com/88248797/201135174-af096836-1330-49ae-a3ed-830d2d27ed25.png)

Reply:
It's a property in the font itself, it has margin-bottom built in it?

disabling font property in this selector fix the weird alignment

```css
button, input, optgroup, select, textarea {
    color: inherit;
    /* font: inherit; */
    margin: 0;
  ```

Reply:
Sure, we could leave it as you have it.